src/andamio/data/course.json

```diff
@@ -50,7 +50,7 @@
             "slts": [
                 {
                     "id": "102.1",
-                    "slt": "I can use a Mesh Provider to get information about the blockchain."
+                    "slt": "I can use a Mesh Providers to get information about the blockchain."
```

Comment:
Does that mean fetcher providers like maestro or blockfrost? If so, the word `Mesh Providers` looks a bit confusing, coz we use the `MeshProvider` to wrap the app.ts.

Suggest change to something like
- blockchain indexing Providers
- external Providers
- fetcher Providers

Reply:
or just a Provider from Mesh